FizzBuzz_wFunction.js

```diff
@@ -1,15 +1,44 @@
-function isFizzBuzz (x){
-  if ( x % 15 === 0){
-    return "FIZZBUZZ"
+function isFizzBuzz_3_5 (x){
+  if ( x % 15 === 0){;
   } else if ( x % 5 === 0){
-    return "BUZZ"
+    return "BUZZ";
   } else if ( x % 3 === 0){
-    return "FIZZ"
+    return "FIZZ";
   } else {
-    return x
+    return x;
+  }
+}
+
+function isFizzBuzz_4_7 (x){
+  if ( x % 28 === 0){
+    return "FIZZBUZZ";
+  } else if ( x % 7 === 0){
+    return "BUZZ";
+  } else if ( x % 4 === 0){
+    return "FIZZ";
+  } else {
+    return x;
+  }
+}
+
+/*
+ *  returns FIZZ for any number x divisible by num1
+ *  returns BUZZ for any number x divisible by num2
+ *  returns FIZZBUZZ for any number x diviible by both num1 && num2
+ */
+
+function isFizzBuzz_x_y(x,num1,num2){
```

Comment:
ok this works obviously. so well done! 🎉 

But I wanted to get to the topic of functions returning other functions. Can you change this in the following way:

1. write a new function `makeFizzBuzzFunction(num1, num2)` that returns a **function**.
2. this returned function should behave like one of the `isFizzBuzz_num1_num2` functions.
3. in the for loop, you first create the correct isFizzBuzz function, then call it. 
    ```
    for (x = 1; x <= 100; x++) {
        var fizzBuzz_3_7 = makeFizzBuzzFunction(3, 7);
        console.log ( fizzBuzz_3_7(x) );
     }
    ```